WorkloadManagementSystem/Executor/JobPath.py

```diff
@@ -53,7 +53,7 @@ def __executeVOPlugin(self, voPlugin, jobState):
       modInstance = self.__voPlugins[voPlugin](argsDict)
       result = modInstance.execute()
     except Exception as excp:
-      self.jobLog.exception("Excp while executing %s" % voPlugin)
+      self.jobLog.exception("Excp while executing", voPlugin)
```

Comment:
you may want to print the exception too ?

Reply:
the stacktrace of the exception will always be printed.

Reply:
ah true !